content/asp.net/getting-started/views.md

```diff
@@ -1,20 +1,121 @@
+
+``` csharp
+@section scripts{
+    <!-- add your JavaScript sources here -->
+}
+```
+
+## Using the Data Passed to Our Views
+
+Now that we're starting to see how 
+
+In the sample project we have a controller called `PersonController` with an action method on it called `Index`. Index accepts an integer called `id` as a parameter, retrieves the requested person from the service and returns with a call to the `View` helper method, passing in the person that was resolved from the service. 
+
+``` csharp
+public IActionResult Index(int id)
+{
+    var person = _personService.GetById(id);
+    return View(person);
+}
+```
+
+We use the above code with the view we discussed at the start of this document, which we'll revisit now.
+
+```html
+@model MvcBasics.Models.Person
+
+<h1>Person Details</h1>
+<p>@Model.FirstName @Model.LastName</p>
+```
+
+In the first line of our document we instruct the view engine to use the `Person` class from our project. The person is set in the `Model` property for us at runtime by virtue of us having passed an instance of a person to the `View` helper method in the controller.  To access the properties of our `Person` object, we use the `@Model.FirstName` notation.
+
+There is much more to take advantage of in the Razor view engine and further reading is available at the official documentation, linked to in the "Next Steps" section at the end of the document.
 
-See the [Issue](https://github.com/dotnet/training-tutorials/issues/65) to claim this lesson and/or view what should be included in it.
+## Incorporating Partial Views
 
-## First Header
+Partial views are incomplete on their own, but can be used like components to help build a page. You might break out your site's menu or part of your footer so that it can be used conditionally depending on other runtime factors. 
 
-Start the lesson here.
+You can inject the results of a rendered partial view into your page by using the HTML helper for partials.
 
-...
+``` csharp
+@Html.Partial("_CopyrightStamp")
+```
 
+A partial view has no layout applied to it, so HTML elements in it should be crafted in a way that respect the design frameworks of the pages they will appear in.  
 
 ## Next Steps
 
-Give the reader some additional exercises/tasks they can perform to try out what they've just learned.
\ No newline at end of file
+For more information on working with Razor views check out the official [ASP.NET documentation](https://docs.asp.net/en/latest/mvc/views/index.html), or these related resources in this tutorial:
```

Comment:
Most lessons give the user some "homework". The additional reading/info, is good, too, but before that can you ask the user to use some of what they've just learned in an applied manner? Ask them create a new view or layout (or both) to see how they work together, for instance.